docs/adopted-rules.md

```diff
@@ -0,0 +1,122 @@
+# Adopted Rules
+
+## Weapon and Armor Levels
+(*Source: [Fate Core](https://fate-srd.com/fate-core/more-examples-extras#weapon-and-armor-ratings)*)
+
+Physical weapons and armor are not typically tracked as resources in Fate. 
+
+Instead, "Weapon" and "Armor" refer to a type of Extra that a character can have. Taking Weapon or Armor Extra grants a passive modifier to damage calculation. Weapon adds to damage dealt, and Armor reduces damage taken. The modifier is equal to the rating Level. 
+
+### Rolling with Weapon and Armor
+
+Weapon and Armor are factored separately from the roll's Shifts and do not apply when the Attack roll outright fails. 
+
+> A character with \[Weapon: 1\] rolls a 5 to Attack against a Defend of 3, yielding 2 Shifts. 
+> 
+> Because this Attack succeeds, an additional 1 point of Weapon damage is dealt for a total of 3. However, the Attacker does not Succeed with Style because they only rolled 2 Shifts.
+
+**Weapons** *do* apply to Tie rolls. However, the damage dealt by Weapons replaces the Boost that an Attacker would receive. 
+
+**Armor** has the potential to negate all damage shifts. In this case, the Attacker receives a Boost just as if it were a Tie.
+
+| Action | Attack Failure | Attack Tie | Attack Succeed |
+|--------|------|------|------|
+| Weapon | Attack is prevented. Do not apply Weapon damage. | Apply only Weapon damage, but do not create a Boost. | Deal Shifts of damage, then add damage equal to Weapon Level. | 
+| Armor | No damage is taken. No Boosts are made.  | If Attacker has Weapon, subtract Armor from damage received. Otherwise, Attacker gets a Boost. | Subtract Armor level from damage. If damage is now 0, Defender Boost. |
+
+Effectively, Weapon tags mean you always deal damage even if you tie, while Armor tags increase the threshold at which you would tie.
+
+### Usage Tips
+
+As with all Extras, Weapon and Armor are best given to a character for a narrative reason. For example, a character with an Aspect reflecting Divinity may have Armor to reflect their power level over mortals. A Stunt reflecting use of a special literal weapon may grant a Weapon level. Bosses may gain levels in Weapon or Armor to reflect a change in phase or form.
+
+Weapon and Armor levels are very powerful and should be assigned conservatively. Level 1 of something may sound low, but Level 2 is effectively a perpetual Free Invoke. Level 3 is extraordinarily powerful and should be balanced as such.
+
+Weapon and Armor are our currently recommended mechanics for damage factors. Further options can be explored in [The Fate System Toolkit](https://fate-srd.com/fate-system-toolkit/weapons-and-armor-alternatives), but have not yet been reviewed for use in Fate/moon.
+
+## Obstacles
```

Comment:
Toss an example obstacle on here to make it super clear!

Reply:
Oops forgot that obstacles are the superset. IGNORE ME 😅

Reply:
Even though each type of obstacle has its own examples?